<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>960</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton::pressed
{
	border-image:url(:/IMG/clicked.png);
}
QPushButton::checked
{
	border-image:url(:/IMG/clicked.png);
};
border-image:url(:/IMG/background.jpg);
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_minimize">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_maximize">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>ㅁ</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_x">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active/>
          <inactive/>
          <disabled/>
         </palette>
        </property>
        <property name="font">
         <font>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane{ border:none}

QTabBar::tab{height:30px; width: 100px;border-image:url(:/IMG/background.jpg);}

QScrollBar:vertical
{
	border: 0px solid #999999;
	background:none;
    width: 10px;
    margin: 0px 0px 0px 0px;
	 
}
QScrollBar::handle:vertical
 {
	border: 0px solid red;
	border-radius: 4px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.313514 rgba(174, 134, 80, 255), stop:1 rgba(220, 200, 155, 255));
 }
QScrollBar::add-line:vertical {
     height: 0px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     height: 0px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar:horizontal
{
	border: 0px solid #999999;
	background:none;
    height: 25px;
    margin: 0px 0px 0px 0px;
	 
}
QScrollBar::handle:horizontal
 {
	border: 0px solid red;
	border-radius: 4px;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.313514 rgba(174, 134, 80, 255), stop:1 rgba(220, 200, 155, 255));
 }
QScrollBar::add-line:horizontal {
     height: 0px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:horizontal {
     height: 0px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }

QListView::item { outline: none; }
QListView::item:selected:active:hover{
    background-color:rgba(149, 102, 27,70%); color: white;
}
QListView::item:selected:active:!hover{
    background-color: rgba(149, 102, 27,70%); color: white;
}
QListView::item:selected:!active{
    background-color:rgba(149, 102, 27,70%); color: white;
}
QListView::item:!selected:hover{
    background-color:rgba(149, 102, 27,70%); color: white;
};


border:none;

</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tab_item">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>아이템</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="spacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,3,2,3,3,3">
              <item>
               <widget class="QLabel" name="label_name">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>휴먼편지체</family>
                  <pointsize>24</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::DefaultContextMenu</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_img">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>125</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="list_explanation">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>태 나무</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">padding:8px 8px;
</string>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideMiddle</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="list_material">
                <property name="font">
                 <font>
                  <family>휴먼옛체</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">padding: 8px 8px;</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="text_etc">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>휴먼가는샘체</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">padding:8px;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="list_item">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>휴먼옛체</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">padding:8px 8px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>휴먼중간팸체</family>
                <pointsize>32</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>검색</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_button" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
        <item>
         <widget class="QPushButton" name="pushButton_1">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>도구</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>조명</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>생존</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>음식</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>과학</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>전투</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>구조물</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>가공</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_14">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
